src/NServiceBus.PersistenceTests/RequiresExtensionsForPersistenceTestsConfiguration.cs

```diff
@@ -10,10 +10,6 @@ public static void RequiresDtcSupport(this IPersistenceTestsConfiguration config
             {
                 Assert.Ignore("Ignoring this test because it requires DTC transaction support from persister.");
             }
-
-#if NET
-            Assert.Ignore(".NET does not support distributed transactions.");
```

Comment:
No, because .NET does support distributed transactions now when running on Windows.

We can either bake an OS check into these Requires checks at the Core level, or we can leave that up to the downstreams to implement the check in their transport/persistence `SupportsDtc` checks.

I'm inclined to leave it alone here and modify downstreams instead.